nautobot_device_lifecycle_mgmt/forms.py

```diff
@@ -913,11 +919,7 @@ class ProviderLCMFilterForm(BootstrapMixin, forms.ModelForm):
 
     q = forms.CharField(required=False, label="Search")
     name = forms.CharField(required=False)
-    country = forms.ChoiceField(
-        widget=StaticSelect2,
-        required=False,
-        choices=add_blank_choice(CountryCodes.CHOICES),
-    )
+    country = forms.MultipleChoiceField(required=False, choices=CountryCodes.CHOICES, widget=StaticSelect2Multiple())
```

Comment:
why remove `add_blank_choice`?

Reply:
I copied and pasted something with fixed and that wasn't in there. I added it back.

Reply:
@itdependsnetworks I don't think `add_blank_choice` does anything when used with `MultipleChoiceField`? We previously had `ChoiceField` here hence why `add_blank_choice` was used.

Reply:
I tested both ways and there isn't a noticeable difference when selecting the filter.

Reply:
ok, makes sense, yet another pylint nautobot use case to consider.